test/Microsoft.DocAsCode.Metadata.ManagedReference.Tests/GenerateMetadataFromAssemblyTest.cs

```diff
@@ -47,5 +61,38 @@ public void TestGenerateMetadataFromAssembly()
                 Assert.Equal(new[] { "System.Object", "CatLibrary.Cat{CatLibrary.Dog{System.String},System.Int32}" }, @class.Inheritance);
             }
         }
+
+        [Fact]
+        public void TestGenerateMetadataFromAssemblyWithReferences()
```

Comment:
>TestGenerateMetadataFromAssemblyWithReferences [](start = 20, length = 46)

Could you also add some test cases in `GenerateMetadataFromCSUnitTest.cs` and `GenerateMetadataFromVBUnitTest.cs`?

Reply:
Sure, will do it